app/assets/javascripts/models/list.js

```diff
@@ -0,0 +1,42 @@
+var List = function(info){
+  this.title = info.title
+  this.id = info.id
+}
+
+List.fetch = function(){
+  // saving the ajax request to a local variable
+    var request = $.getJSON("http://localhost:3000/lists")
```

Comment:
remove the protocol, host, and port to make it easier to deploy this application in the future.